BDArmory/GameModes/ModuleSpaceFriction.cs

```diff
@@ -159,6 +159,7 @@ public void FixedUpdate()
                             maxVelocity = VAI.MaxSpeed;
 
                         var speedFraction = (float)part.vessel.speed / maxVelocity;
+                        if (speedFraction > 1) speedFraction = Mathf.Max(2, speedFraction);
```

Comment:
Is this meant to be `Max` or `Min` here? With `Max`, the `speedFraction` jumps to 2 as soon as `speedFraction` exceeds 1.

Reply:
max; it's intended as a sanity check in case RCS or something pushes craft above the maxSpeed, to boost the drag to bring things back down under maxSpeed, as going over it causes the AI to sort of stop working in space in some condifitons

Reply:
OK, cool.